parsers/NA.py

```diff
@@ -52,6 +50,31 @@ def get_text_from_image(link, expected_size, new_size, logger, session=None):
     return text
 
 
+def timestamp_checker(session=None, logger=None):
+
+    scada_info = get_text_from_image(session=session, link=timestamp_link, \
+                                    expected_size=(600,20), new_size=(1200,40), \
+                                    logger=logger)
+
+    timestamp = scada_info.split(':', 1)[1]
+
+    try:
+        scada_time = arrow.get(timestamp, ' DD/MM/YYYY HH:mm:ss')
+    except arrow.parser.ParserError as e:
+        logger.warning('Namibia scada timestamp cannot be read, got {}.'.format(timestamp))
+        return True
+
+    data_time = scada_time.replace(tzinfo='Africa/Windhoek')
+    current_time = arrow.now('Africa/Windhoek')
+    diff = current_time - data_time
+
+    # Need to be sure we don't get old data if image stops updating.
+    if diff.seconds > 3600:
+        raise ValueError('Namibia scada data is too old to use, data is {} hours old.'.format(diff.seconds/3600))
+    else:
+        return True
```

Comment:
why does `check_timestamp` return a value? It never seems used.

Reply:
Agree that the return in the else statement can go, the one in the try/except is used to exit if the timestamp can't be read.